### ISSUE TITLE
Vertex count check missing for undirected

### ISSUE THREAD

**Issue by Willem3141** [graphchecker/moodle-qtype_graphchecker#103]
In GitLab by @Willem3141 on Aug 10, 2020, 15:26

Therefore this old question stopped working:

```
[{"module":"basic","method":"vertex_count","arguments":{"expected":"1"}}]
```

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on Aug 10, 2020, 15:26

changed the description

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on Aug 10, 2020, 15:33

closed via commit 506e6342bb1aca4700befad7edbce2d177b0fffc

**Comment by Willem3141** [comment]
In GitLab by @Taki on Aug 21, 2020, 10:21

mentioned in commit 506e6342bb1aca4700befad7edbce2d177b0fffc
